markdown-mode.el

```diff
@@ -3394,23 +3395,6 @@ the buffer)."
 (defun markdown-match-yaml-metadata-key (last)
   (markdown-match-propertized-text 'markdown-metadata-key last))
 
-(defun markdown-match-wiki-link (last)
```

Comment:
Please don't remove public interfaces, this causes breaking backward compatibility